app/helpers/application_helper.rb

```diff
@@ -409,6 +404,15 @@ def edit_inline(object, property, options = {})
     end
   end
 
+  def documentation_url(section = "")
+    if section.empty?
+      base_url = "http://www.theforeman.org/documentation.html##{SETTINGS[:version].short}"
+    else
+      base_url = "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#" + section
+    end
+    base_url
```

Comment:
No need to create a variable, 

```ruby
def documentation_url(section = '')
   if section.empty?
     "http://www.theforeman.org/documentation.html##{SETTINGS[:version].short}"
   else
     "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#" + section
  end
end
```

is 100% equivalent without creating any local variable